controlnext/templates/controlnext/selectbasin.html

```diff
@@ -0,0 +1,36 @@
+{% extends 'controlnext/basin_detail.html' %}
+{% load i18n %}
+
+{% block content %}
+<div class="container" style="height:150px">
+</div>
+<div class="container">
+    <div class="row">
+      <div class="col-md-4">
+      </div>
+      <div class="col-md-4">
+        <div class="panel panel-default">
+
+          <div class="panel-heading">Meerdere bassins beschikbaar. Kies het
+              gewenste Bassin.</div>
+
+          <div class="btn-group-vertical panel-body" role="group"
+               style="width:100%;">
+            {% for random_url_slug, basin_name in view.buttons %}
+              <a class="btn btn-default"
+                 href="{% url controlnext-basin random_url_slug %}">{{ basin_name }}</a>
+            {% endfor %}
+          </div>
+        </div>
+        <a class="btn btn-info" role="button" style="width:100%;"
+           href="/logout/?next=/">logout</a>
+      </div>
+    </div>
+</div>
+{% endblock content %}
+
+{% block constants %}
```

Comment:
Dit kan een goede oplossing zijn, maar kijk even een minuutje of er een andere oplossing is.

Volgens mij heb je nu voor dit ene template twee blocks in het hoofdtemplate gezet om ze hier uit te kunnen zetten. Even kijken of het anders kan. (Of dat het prima is, is ook goed).